parsers/NTESMO.py

```diff
@@ -68,6 +68,11 @@ class Generator(TypedDict):
     status_forcelist=[500, 502, 503, 504],
 )
 
+# Default behaviour for datetime
+def get_default_datetime_if_not_None(target_datetime: datetime) -> datetime:
+    if target_datetime is None:
+        return datetime.now(tz=AUSTRALIA_TZ) - timedelta(days=2)
```

Comment:
```suggestion
        return datetime.now(tz=AUSTRALIA_TZ) - timedelta(days=DELAY)
```
There's a new delay variable to be used here.

Reply:
You're right - I will create a `DAY_DELAY` variable, and set it equal to 5, given the comment at the top of the file that "data is being published after 5 days"